target/linux/ar71xx/mikrotik/config-default

```diff
@@ -89,6 +89,7 @@ CONFIG_ATH79_MACH_RBSXTLITE=y
 # CONFIG_ATH79_MACH_TL_WR1041N_V2 is not set
 # CONFIG_ATH79_MACH_TL_WR1043ND is not set
 # CONFIG_ATH79_MACH_TL_WR1043ND_V2 is not set
+# CONFIG_ATH79_MACH_TL_WR1043ND_V4 is not set
```

Comment:
AFAIK, these files shouldn't be changed manually. Instead, `make kernel_menuconfig CONGI_TARGET=subtarget` with save on exit should be used (after selecting _Generic devices with NAND flash_ in `make menuconfig`). That will pickup, refresh and update related kernel config-default.

Anyway, if seems that kernel configs for subtargets in ar71xx need general refresh and cleanups. Ex. there are many non-NAND boards enabled in ...nand/config-default.